### PR TITLE
Constrain transactions checks

### PR DESCRIPTION
**Pull request by ylavoie**
- Fix the AP/AR use of invalid field reference
- Point to proper table in orphaned transaction included files

### CODE REVIEW

File: sql/changes/1.8/constrain-transaction-tables.sql.checks.pl [hunk 1]
```diff
@@ -352,11 +352,11 @@ package _18_upgrade_checks;
 the database anymore. As a result, the files attached to these transactions
 cannot remain connected. Two options are available:
 
-1. 'Delete' the files listed below
```
Comment (ehuelsmann) [inline sql/changes/1.8/constrain-transaction-tables.sql.checks.pl:355]: This whitespace was there with intent: it forces a new line within the same paragraph. Could you undo the removal, please?

File: sql/changes/1.8/constrain-transaction-tables.sql.checks.pl [hunk 2]
```diff
@@ -352,11 +352,11 @@ package _18_upgrade_checks;
 the database anymore. As a result, the files attached to these transactions
 cannot remain connected. Two options are available:
 
-1. 'Delete' the files listed below  
+1. 'Delete' the files listed below
    This action can't be undone. As the files aren't available in the
    application, due to the fact that they are attached to orphaned
    transactions, this may not make a difference.
-2. 'Copy' the files to the 'incoming' queue
```
Comment (ehuelsmann) [inline sql/changes/1.8/constrain-transaction-tables.sql.checks.pl:359]: (same here)

File: sql/changes/1.8/constrain-transaction-tables.sql.checks.pl (hunk 1 above)
Comment (ylavoie) [reply sql/changes/1.8/constrain-transaction-tables.sql.checks.pl]: Fixed.

File: sql/changes/1.8/constrain-transaction-tables.sql.checks.pl (hunk 2 above)
Comment (ylavoie) [reply sql/changes/1.8/constrain-transaction-tables.sql.checks.pl]: Fixed, using trailing spaces as markdown is not ideal.

**Pull request merged**